app/views/events/_form.html.erb

```diff
@@ -1,49 +1,34 @@
 <%= form_with(model: event, local: true) do |form| %>
-  <% if event.errors.any? %>
-    <div id="error_explanation">
-      <h2><%= pluralize(event.errors.count, "error") %> prohibited this event from being saved:</h2>
+  <%= render 'layouts/errors', errors: event.errors %>
 
-      <ul>
-        <% event.errors.full_messages.each do |message| %>
-          <li><%= message %></li>
-        <% end %>
-      </ul>
-    </div>
-  <% end %>
-
-  <div class="input-group mb-12 mb-2">
-    <div class="input-group-prepend">
-      <%= form.label :name, "Esemény neve", class: "input-group-text" %>
-
-    </div>
+  <div class="mb-2">
+    <%= form.label :name, "Esemény neve", class: "form-label" %>
     <%= form.text_field :name, class: "form-control" %>
   </div>
 
-  <div class="input-group mb-12 mb-2">
-    <div class="input-group-prepend">
-      <%= form.label :date, class: "input-group-text" %>
-
+  <div class="mb-2 align-items-center">
+    <%= form.label :date, class: "form-label" %>
+    <div class="form-control">
+      <%= form.datetime_select :date, {}, class: "" %>
     </div>
-    <%= form.datetime_select :date, class: "form-control" %>
   </div>
 
-  <div class="input-group mb-12 mb-2">
-    <div class="input-group-prepend">
-      <%= form.label :event_type_id, "Jelentkező űrlap", class: "input-group-text" %>
-
-    </div>
+  <div class="mb-5">
+    <%= form.label :event_type_id, "Jelentkező űrlap", class: "form-label" %>
     <%= form.select :event_type_id,
-                    options_for_select(@event_types = EventType.all.pluck(:name, :id)),
-                    include_blank: "Nincs jelentkezés",
-                    class:         "form-control" %>
+      EventType.all.collect {|et| [ et.name, et.id ] },
```

Comment:
Is there a reason why use `EventType.all.collect {|et| [ et.name, et.id ] }` instead of the shorter `EventType.pluck(:name, :id)`. AFAIK pluck is supposed to be a bit more efficient too.

Reply:
Yep, this is what the documentation suggested best, and this way if there's already an eventtype chosen for an event, the chosen one will be loaded into the <select> element. With pluck this wouldn't work.

Reply:
But gonna look into it, don't know why it shouldn't work with pluck

Reply:
So the problem was not the pluck which caused not to preload the setting from before, but that there were no curly brackets around the include_blank option (all options should be in one hash)

Reply:
# :ballot_box_with_check: